### PR TITLE
fix full vehicle, hax, add prop spawn and interactions

### PR DESCRIPTION
**Pull request by avcio9**
vehicle damage and destroy worked well already, still have to fix interactionstring on mounted cars spawned by server.

Some props dont send interactionstrings or you have to find a weird angle to trigger it (they may be covered by client-spawned objects) 
We have to find a way to despawn the default spawned objects

### CODE REVIEW

**Review by QuentinGruber — APPROVED** [review]

**Comment by QuentinGruber** [comment]
@avcio9 About despawning, there is a packet called despawnNpc if i remember well but no idea on how to get the id of the client side spawned object

File: src/servers/ZoneServer/zoneserver.ts [hunk 1]
```diff
@@ -536,7 +546,7 @@ export class ZoneServer extends EventEmitter {
     ];
     this._dummySelf = require("../../../data/2015/sampleData/sendself.json"); // dummy this._dummySelf
     if (String(client.character.characterId) === "0x0000000000000001") {
-      // for fun 🤠
+      // for fun ­Ъца
```
Comment (QuentinGruber) [inline src/servers/ZoneServer/zoneserver.ts:549]: What does it mean ? 😀

File: src/servers/ZoneServer/zoneserver.ts [hunk 2]
```diff
@@ -795,7 +806,7 @@ export class ZoneServer extends EventEmitter {
 
       this.sendData(
         client,
-        "PlayerUpdate.RemovePlayerGracefully",
```
Comment (QuentinGruber) [inline src/servers/ZoneServer/zoneserver.ts:798]: Why changing that? With this packet the entity is still in the game i don't think we should use it

**Review by QuentinGruber — CHANGES_REQUESTED** [review]

File: src/servers/ZoneServer/zoneserver.ts (hunk 1 above)
Comment (avcio9) [reply src/servers/ZoneServer/zoneserver.ts:549]: Pretty :P

File: src/servers/ZoneServer/zoneserver.ts (hunk 2 above)
Comment (avcio9) [reply src/servers/ZoneServer/zoneserver.ts:798]: RemovePlayer packet for less resource usage, if it doesnt work that way ima change it back

**Comment by avcio9** [comment]
> @avcio9 About despawning, there is a packet called despawnNpc if i remember well but no idea on how to get the id of the client side spawned object

All the .zone objects have unique id's in json files, maybe thats the way

**Review by QuentinGruber — APPROVED** [review]

**Pull request merged**